impeller/scene/importer/conversions.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
Ditto question about include guards in this file.

Reply:
Fixed